x-pack/plugins/security_solution/public/management/components/endpoint_responder/get_processes_action.tsx

```diff
@@ -38,34 +39,36 @@ const StyledEuiBasicTable = styled(EuiBasicTable)`
   }
 `;
 
-export const GetRunningProcessesActionResult = memo<
+export const GetProcessesActionResult = memo<
   CommandExecutionComponentProps<
     { comment?: string },
     {
       actionId?: string;
       actionRequestSent?: boolean;
-      completedActionDetails?: ActionDetails<RunningProcessesEntry>;
+      completedActionDetails?: ActionDetails<ProcessesEntry>;
+      apiError?: HttpFetchError;
     },
     EndpointCommandDefinitionMeta
   >
 >(({ command, setStore, store, status, setStatus, ResultComponent }) => {
   const endpointId = command.commandDefinition?.meta?.endpointId;
-  const { actionId, completedActionDetails } = store;
+  const { actionId, completedActionDetails, apiError } = store;
 
   const isPending = status === 'pending';
+  const isError = status === 'error';
   const actionRequestSent = Boolean(store.actionRequestSent);
 
-  const getRunningProcessesApi = useSendGetEndpointRunningProcessesRequest();
+  const getProcessesApi = useSendGetEndpointProcessesRequest();
```

Comment:
Is there a reason we don't want to destructure this object  to `{mutate, data, ...}` etc.